packages/datepicker/src/DatePicker.tsx

```diff
@@ -0,0 +1,117 @@
+import { FunctionComponent } from 'react';
+import DatePicker from 'react-datepicker';
+import { FusionDatePickerProps } from './types';
+import FusionDatePickerInput from './DatePickerInput';
+import FusionDatePickerHeader from './DatePickerHeader';
+import { makeStyles, createStyles, FusionTheme } from '@equinor/fusion-react-styles';
+
+import 'react-datepicker/dist/react-datepicker.css';
+
+type StyleProps = {
+  fluid?: boolean;
+};
+
+const useStyles = makeStyles<FusionTheme, StyleProps>(
+  () =>
+    createStyles({
+      wrapper: ({ fluid }) => ({
+        width: fluid ? '100%' : 'auto',
+      }),
+    }),
+  { name: 'fusion-datepicker-header' }
+);
+
+export const FusionDatePicker: FunctionComponent<FusionDatePickerProps> = (
+  props: FusionDatePickerProps
+): JSX.Element => {
+  const {
```

Comment:
could you make a const defaultProps outside the component.

const {someprop, someotherProp, ...options} = {...defaultProps, props}
return <DataPicker {...options}  theProp={someprop} />